ote_sdk/ote_sdk/test_suite/training_tests_common.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from typing import Union
+
+from ote_sdk.entities.metrics import Performance, ScoreMetric
+
+# pylint: disable=invalid-name
+
+
+def DEFAULT_FIELD_VALUE_FOR_USING_IN_TEST():
```

Comment:
please follow PEP convention or just use CONSTANTS

Reply:
OMG, Ilya, do you **really** think that it is a good idea to use variables instead of functions that return constant values?

Reply:
Fixed